marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/security/getAuthoritiesTest.sjs

```diff
@@ -0,0 +1,24 @@
+const test = require("/test/test-helper.xqy");
```

Comment:
There are a lot of other scenarios to test here, each as a different user. Switching users is very difficult in marklogic-unit-test though. For that reason, I think this test should be converted into a JUnit test - GetAuthoritiestTest - where each test method uses a different user. We have test-data-hub-developer and test-data-hub-operator already, and I think you can use "admin" too.

Reply:
I thought about it. It is why I added more test cases in the AuthTest.

Reply:
Consider though that we'll likely add a lot more logic to this endpoint. Will it be easier to write tests against SecurityService, where all we need to do is instantiate a SecurityService, or against the Spring MVC layer, where we need to deal with mock request and response objects and a lot of other logic that we don't really care about? 

It's good to have one test against Spring MVC to verify that all the parts are integrated together nicely. But for all the different scenarios that getAuthorities.sjs will handle, it'll be a lot cheaper in the long run to verify those scenarios directly against the endpoint.

Reply:
@rjrudin Can you give me an example of a java test class about how to invoke sjs ?

Reply:
You'd just use the DS-generated one - SecurityService.